packages/bridge-ui-v2/.env.example

```diff
@@ -0,0 +1,66 @@
+# Chain Names
```

Comment:
@jscriptcoder can we do all these with `export`? I'm not sure if it still works fine, if you could please check. 

why? because otherwise this does not support the sort of popular `fish` shell (okay I use it). see here: https://github.com/d1onys1us/dapp-slaps/blob/main/.env.example#L3

Reply:
Seems to work